enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -226,6 +228,10 @@
             ( logs ) -> new File( logs, "security.log" ),
             PATH );
 
+    @Description("Security log level threshold.")
+    public static final Setting<Level> security_log_level = setting( "dbms.security.log_level",
```

Comment:
It would be good to have the setting in enterprise neo4j.conf, commented-out with default value  (even if debug log level is not there)